adaptors_task/student/src/impl.cpp

```diff
@@ -1,8 +1,49 @@
 #include "impl.h"
 #include <vector>
+#include <algorithm>
+#include <stack>
 
 const std::vector<std::pair<char, char>> brackets {
     {'(', ')'},
     {'[', ']'},
     {'{', '}'}
 };
+
+auto findBracket(const char bracket) -> std::vector<std::pair<char, char>>::const_iterator
```

Comment:
1. What is a reason for trailing return type?
2. Please give more descriptive name